test/errors.js

```diff
@@ -7,6 +7,63 @@ const { ErrorWithProps } = GQL
 const { FederatedError } = require('../lib/errors')
 const split = require('split2')
 
+test('errors - incorrect query error', async (t) => {
+  const schema = `
+    type Query {
+      error: String
+      successful: String
+    }
+  `
+
+  const resolvers = {
+    Query: {
+      error (err) {
+        console.log(JSON.stringify(err))
+        throw new ErrorWithProps('Error', { code: 'ERROR', additional: 'information', other: 'data' })
+      },
+      successful () {
+        return 'Runs OK'
+      }
+    }
+  }
+
+  const app = Fastify()
+
+  app.register(GQL, {
+    schema,
+    resolvers
+  })
+
+  await app.ready()
+
+  const res = await app.inject({
+    method: 'GET',
+    url: '/graphql?querys={error,successful}'
```

Comment:
You have a typo here. You are performing a non-existent query. After you perform the correct query, the extensions are correctly reported.

```suggestion
    url: '/graphql?query={error,successful}'
```